test/Kubernetes/KubernetesApiExtensionsTests.cs

```diff
@@ -54,4 +108,15 @@ public void WithAnnotations(bool doReplace)
         Assert.Equal(expected, newJob.Metadata.Annotations);
         Assert.Equal(expected, newJob.Spec.Template.Metadata.Annotations);
     }
-}
+
+    [Fact]
+    public void WithPodPolicyFailureDisruptionTarget_AddsIgnoreRule()
+    {
+        var job = new V1Job();
+
+        var result = job.WithPodPolicyFailureDisruptionTarget();
+
+        var actions = result.Spec.PodFailurePolicy.Rules.Select(rule => rule.Action).ToList();
+        Assert.Contains("Ignore", actions);
```

Comment:
is there a better way to check that disruption target is also there?

Reply:
I've made a stricter assertion checking the "Disruption target" too in [d21279a](https://github.com/SneaksAndData/esd-services-sdk/pull/75/commits/d21279af3c03081c2504d45213df2471774025cd)
I hope this is better 👍